networking/synthetic/scripts/tuning.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+. /usr/lib/tuned/functions
+
+start() {
+    tuna -c22 -q p5p1 -m -x
```

Comment:
I see you are replacing p5p1 later...its just a little misleading to see it here; perhaps make this instead of p5p1 make it ```benchmark_interface``` ?